### DISCUSSION TITLE
Help me with the MDX component issue

### DISCUSSION THREAD

**Discussion by PythonHacker24** [fuma-nama/fumadocs#788] [Q&A]
I am not a seasoned web developer, and I just need help with this MDX thing. It would be great if you could elaborate on the steps.

![Screenshot 2024-09-02 at 17 55 28](https://github.com/user-attachments/assets/f915a6ae-e934-4d71-bbf8-c177a7718282)

This is where I am stuck ☝️
page.tsx content would help me a lot 

This is the error I want to get solve:

<img width="1003" alt="Screenshot 2024-09-02 at 17 58 19" src="https://github.com/user-attachments/assets/189fb10c-57dd-4e1b-ad2a-7263ef80fb34">

Thanks

**Comment by fuma-nama** [comment]
In your page.tsx, there should be a MDX/page.data.body component that accepts the components as a prop, and render your MDX content. Pass your APIPage component to it

**Comment by PythonHacker24** [comment]
Would you please send me a page.tsx? That would do it. I have followed instructions from the docs, and I think this is the last error I will face before getting my docs up. I am not a web developers so I don't know much about it.

**Reply by fuma-nama** [reply]
It looks like

https://fumadocs.vercel.app/docs/ui/manual-installation#page

You can modify the components prop to pass the APIPage component

**Comment by Gitstar-OC** [comment]
Do you need more help? @PythonHacker24, I can help

**Reply by PythonHacker24** [reply]
Hey @Gitstar-OC, thanks for asking. I am done with the project for now.

**Reply by Gitstar-OC** [reply]
Hey, I guess you messaged me on linkedin ? Sorry I didn't see the message. Is there any problem you are facing right now ? I would love to help or even contribute to the very own thing you are building!
